packages/host/app/components/operator-mode/container.gts

```diff
@@ -612,22 +674,28 @@ export default class OperatorModeContainer extends Component<Signature> {
             </button>
           {{/if}}
         </div>
-
-        {{#if this.isChatVisible}}
-          <ChatSidebar @onClose={{this.toggleChat}} />
-        {{else}}
-          <IconButton
-            data-test-open-chat
-            class='chat-btn'
-            @icon='sparkle'
-            @width='30px'
-            @height='30px'
-            {{on 'click' this.toggleChat}}
-            style={{cssVar
-              boxel-icon-button-width='50px'
-              boxel-icon-button-height='50px'
-            }}
-          />
+        {{#if APP.experimentalAIEnabled}}
+          {{#if this.isChatVisible}}
```

Comment:
i think this more clearly expressed as an `and`. you can either make a getter for that or use the `and` template helper from boxel-ui

Reply:
you never did this.... why not?

Reply:
I can make that change, though looking at it now there's 3 branches.

Enabled & open - show chat

Enabled & closed - show button

Not enabled - show neither

We could combine these but we'll need at least two getters rather than a single binary.